### DISCUSSION TITLE
Lyrics provider

### DISCUSSION THREAD

**Discussion by ABHIRAJ770** [fast4x/RiMusic#1610] [Q&A]
How do i change the lyrics provider. Like I remember having to choose a provider and i chose lrclib.net not knowing anything(which is still don't know) and i think many songs doesn't have lyrics. So i want to change it. Also someone recommend me which is the best provider

**Comment by fast4x** [comment]
Actually lyrics providers are YT, Kugou and LrcLib automatically selected.

**Reply by ABHIRAJ770** [reply]
No I remember there were many providers to choose from. But I can't find the that list now

**Comment by ABHIRAJ770** [comment]
> Actually lyrics providers are YT, Kugou and LrcLib automatically selected.

Even if there are automatically selected I want to select yt manually

**Comment by fast4x** [comment]
Will be developed in the future.

**Comment by NEVARLeVrai** [comment]
@fast4x Is it possible to have dynamic lyrics? Like Spotify or like Apple music?

**Comment by NEVARLeVrai** [comment]
@fast4x Is it possible to have dynamic lyrics? Like Apple music?

**Reply by NEVARLeVrai** [reply]
https://github.com/surfbryce/beautiful-lyrics

It's a plugin for spicetify, if you can maybe help you out to make this.

**Reply by OrangeZXZ** [reply]
> @fast4x Is it possible to have dynamic lyrics? Like Spotify or like Apple music?

What do you mean by Dynamic lyrics? Normally it is already in if you refer to lyrics that move with the song

**Reply by OrangeZXZ** [reply]
> Maybe this can help you ?
> 
> https://github.com/surfbryce/beautiful-lyrics
> 
> It's a plugin for spicetify, if you can maybe help you out to make this.

Oh I see i don't think this is possible with the lyrics provider that we have right now

**Reply by NEVARLeVrai** [reply]
> > Maybe this can help you ?
> > 
> > https://github.com/surfbryce/beautiful-lyrics
> > 
> > It's a plugin for spicetify, if you can maybe help you out to make this.
> 
> Oh I see i don't think this is possible with the lyrics provider that we have right now


He can look in the project if it gives the service that they use

**Reply by fast4x** [reply]
I think is not compatible for now

**Reply by ALMIGHTY-VC** [reply]
> > > Maybe this can help you ?
> > > https://github.com/surfbryce/beautiful-lyrics
> > > It's a plugin for spicetify, if you can maybe help you out to make this.
> > 
> > 
> > Oh I see i don't think this is possible with the lyrics provider that we have right now
> 
> He can look in the project if it gives the service that they use

I asked the dev to create similae one for foss apps he said it depends entirely on spotify and uses its service not suitable for an app like ri

**Reply by ALMIGHTY-VC** [reply]
Foss apps means for foss app based on yt music

**Reply by ALMIGHTY-VC** [reply]
> https://github.com/surfbryce/beautiful-lyrics
> 
> It's a plugin for spicetify, if you can maybe help you out to make this.

Well even though its unsupported yeah but id love to see the words sync option like beautiful music in its video showcasing improved cards page
